tangelo/linq/simulator.py

```diff
@@ -26,515 +26,35 @@
 (e.g a number of shots is required).
 
 Some backends may only support a subset of the above. This information is
-contained in a separate data-structure.
+contained in backend_info.
```

Comment:
backend_info does not exist here, so maybe specify "the backend_info variable under the linq module."